addon/components/logo-maker.hbs

```diff
@@ -0,0 +1,31 @@
+<div class="fx-col fx-gap-px-6 logo-maker" data-control-name="logo-editor-mode-icon">
```

Comment:
does this data-control-name still make sense ?
The wording was done based on publishr's original component I think.

Maybe you should add the splattributes and remove the data-control-name from here.
This way you can add it to publishr if needed by the tests in that project